src/e2e/java/teammates/e2e/util/JMeterConfig.java

```diff
@@ -122,26 +135,64 @@ public HashTree createTestPlan() {
         onceOnlyController.setProperty(TestElement.TEST_CLASS, OnceOnlyController.class.getName());
         onceOnlyController.setProperty(TestElement.GUI_CLASS, OnceOnlyControllerGui.class.getName());
 
+        // Regex Extractor for CSRF token
+        RegexExtractor regexExtractor = null;
+        if (isPostEndpoint) {
```

Comment:
CSRF token is needed for non-safe HTTP methods, which include `POST`, `PUT`, and `DELETE`.